app/views/reciipes/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="recipelist-container">
+<h1>Recipes</h1>
+  <button class="addrecipe-btn">
+  <%= link_to "New recipe", new_reciipe_path %>
+  </button>
+  </div>
+
+<div id="reciipes">
+  <% @reciipes.each do |reciipe| %>
+    <div class="reciipe-all">
+    <div class="recipe-title">
+   <p> <%= reciipe.name %> </p>
+    <%= button_to "Remove", reciipe_path(reciipe.id), method: :delete %>
+    </div>
+    <p>  <%= reciipe.description %> </p>
+    <p>
+      <%= link_to "Show this reciipe", reciipe %>
```

Comment:
- [ OPTIONAL ] I highly recommend adding beside delete action the modify action so if the user want to modify the 
ingrediant  so he could be able to edit it as shown in the design ⭐🚀💯

- Your design
![image](https://github.com/MayPyone/Recipe-app/assets/58553711/75d13999-2378-4ca0-ab42-75803741c41e)

- Given
![image](https://github.com/MayPyone/Recipe-app/assets/58553711/fe65b9c2-b0d6-4229-8b61-32ea6d56f30e)